commands/blue_green_deploy_command.go

```diff
@@ -49,6 +49,7 @@ func (c *BlueGreenDeployCommand) GetPluginCommand() plugin.Command {
 				util.GetShortOption(applyNamespaceAppNamesOpt):     "(EXPERIMENTAL) Apply namespace to application names: (true, false)",
 				util.GetShortOption(applyNamespaceServiceNamesOpt): "(EXPERIMENTAL) Apply namespace to service names: (true, false)",
 				util.GetShortOption(applyNamespaceAppRoutesOpt):    "(EXPERIMENTAL) Apply namespace to application routes: (true, false)",
+				util.GetShortOption(applyNamespaceAsSuffix):        "(EXPERIMENTAL) Apply namespace at the end of the application name: (true, false)",
```

Comment:
change to 'If present, apply namespace as a suffix rather then a prefix' or 'Apply namespace after names and routes, instead of before' or similar